src/index.js

```diff
@@ -87,26 +83,22 @@ export default {
     },
     
     setFormDidClose(callback) {
-        if (Platform.OS == 'ios') {
-            usabillaEventEmitter.addListener('UBFormDidClose', callback)
-        }
+        usabillaEventEmitter.addListener('UBFormDidClose', callback)
```

Comment:
Yes - lets send for both platforms

Reply:
yes we removed platform specific condition.